src/App.js

```diff
@@ -1,13 +1,15 @@
 import React from "react";
 import Header from "./components/header";
 import Main from "./containers/main";
+import HeaderTop from "./components/header-top";
 
 import "./App.css";
 
 const App = () => (
-  <div>
-    <Header />
+  <div style={{ height: "100%" }}>
+    <HeaderTop />
```

Comment:
This header? does it show on wider screens or mobile screens only. If its' mobile only, you can name it something better like `MobileMenu`

Reply:
Its only on mobile screens. I will change it.